src/ng/components/body/body.html

```diff
@@ -1,7 +1,11 @@
-<tr ng-repeat="$row in $view.body.rows track by $index"
-	 ng-class="{'q-grid-selected': $view.selection.state($row).isSelected}"
-	 q-grid-core:tr>
+<tr ng-repeat-start="$row in $view.body.rows track by $index"
```

Comment:
in my opinion view.body.rows should return list of rows already including details